bin/checkExperimentChanges.sh

```diff
@@ -1,36 +1,130 @@
 #!/usr/bin/env bash
 
+# This script prints a status that will dictate how the workflow routes
+# analysis. 
+
 scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 source $scriptDir/utils.sh
 
 expName=$1
 species=$2
 newConfig=$3
-newSdrf=$4
+newMetaForQuant=$4
```

Comment:
In general, I find bash scripts that rely on positional arguments hard to maintain and debug. In the future, to avoid delaying this, I would suggest that we either prefer proper argument parsing or depend on verbosely named env variables, but avoid in general scripts with large numbers of positional arguments.

Reply:
Yep, sorry @pcm32 - I'm just a bit lazy with the interface when I know the scripts will only be used as a workflow component.